src/atoms/InputLabel/InputLabel.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import { Meta, Story } from "@storybook/react";
+import React from "react";
+import { Label } from ".";
+import { ICustomLabelProps } from "./InputLabel.model";
+import "./inputLabel.scss";
+
+type Props = React.PropsWithChildren<ICustomLabelProps>;
+
+const Template: Story<Props> = ({ label = "INPUT LABEL", ...props }) => (
```

Comment:
Nice use of using default values/destructuring/component composition here imagine doing that a year ago 🎉